src/components/ContactsView.js

```diff
@@ -1,23 +1,36 @@
-import { useEffect, useState } from "react"
-import { useParams } from "react-router-dom"
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
 
 function ContactsView() {
-  const [contact, setContact] = useState(false)
+  const [contact, setContact] = useState(false);
 
   //TODO: Get the contact to load from the params and fetch.
   //With useEffect, load the contact when params changes
   //and update contact state
+  const params = useParams();
+
+  const { id } = params;
+
+  useEffect(async () => {
+    const data = await fetch(`http://localhost:4000/contacts/${id}`);
+    const contact = await data.json();
```

Comment:
clear use of async and await, code is much more readable, I will learn and use this syntax from now on as it is so much cleaner